Tests/SwiftDocCTests/Utility/NearMissTests.swift

```diff
@@ -109,7 +109,6 @@ class NearMissTests: XCTestCase {
             "init(help:)",
             "init(exclusivity:help:)",
             "init(wrappedValue:exclusivity:help:)",
-            "init(wrappedValue:help:)",
```

Comment:
This possible match scored very closely to one of the other matches resulting in it and another match changing order depending on the Swift compiler version. Having two suggestions so close in score to each other could result in further such failures in the future so I opted to remove it to make this test more stable.